src/utils/verifiable-credential/types.ts

```diff
@@ -0,0 +1,64 @@
+export enum TrustworthinessScope {
```

Comment:
Will the directory ever get incoming untrusted VCs that need to be validated?

If yes, may I suggest building types using [superstruct](https://docs.superstructjs.org/) to being with, that way you can validate incoming untrusted VCs with ease.

If no, safe to ignore this comment

Reply:
i think it is not possible, unless people change the JS source code

the VC payload are generate from a builder, and frontend will just pass in some key information, mostly a free text string array to description the action reason

when we sign a message , we also pass a schema for type data sign, when signing the data it will also validate if the data is fit to the schema

but i think it is always good to have validation, we will create a validator in other PR